db/c_level_db.go

```diff
@@ -155,59 +160,83 @@ func (mBatch *cLevelDBBatch) Write() {
 //----------------------------------------
 // Iterator
 
-func (db *CLevelDB) Iterator() Iterator {
+func (db *CLevelDB) Iterator(start, end []byte) Iterator {
```

Comment:
Seek/Prev complicates logic (e.g. will ripple through to the SDK store module).

Seek() isn't used anywhere. It was used in tests because it was the only way to seek.  Now we have start/end instead.

Reply:
I see I see, thanks for pointing that out.